app/src/main/java/com/github/sdp/ratemyepfl/model/items/Classroom.kt

```diff
@@ -14,7 +14,7 @@ data class Classroom(
     companion object {
         fun DocumentSnapshot.toClassroom(): Classroom? {
             return try {
-                Classroom(id)
+                Classroom(id, getString("numRatings")!!.toInt(), getString("avgRating")!!.toDouble())
```

Comment:
Maybe we could handle the null types instead of using a try-catch

Reply:
No values will be null here anyway, we control the database.